src/catalog.ts

```diff
@@ -25,8 +26,10 @@ export class Catalog {
       PREFIX schema: <http://schema.org/>
         SELECT * WHERE {
           ?dataset a schema:Dataset ;
-            schema:distribution ?distribution ;
-            schema:name ?name .
+            schema:name ?name ;
+            schema:creator ?creator ;
+            schema:distribution ?distribution .
+          ?creator schema:identifier ?creatorIdentifier .
```

Comment:
To keep things simple, I support only a single creator for now.

Reply:
Makes sense! We'll deal with multiple creators as soon as a relevant source arrives.